src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -60,22 +60,22 @@ const Embedding = (props) => {
 
   const focusData = useSelector((state) => state.cellInfo.focus);
   const focusedExpression = useSelector((state) => state.genes.expression.data[focusData.key]);
+  const cellSets = useSelector(getCellSets());
   const {
     properties: cellSetProperties,
     hierarchy: cellSetHierarchy,
     hidden: cellSetHidden,
-  } = useSelector(getCellSets());
+  } = cellSets;
 
   const selectedCell = useSelector((state) => state.cellInfo.cellName);
```

Comment:
`cellName` is actually `cellId` right? Is it possible to change `cellName` to `cellId` it so that's it's easier to understand?